### DISCUSSION TITLE
low FPS and So lag when use Dictionary API to translate Label name

### DISCUSSION THREAD

**Discussion by TuyenSD** [ultralytics/yolov5#5934] [Q&A]
i make app for Studying English by detecting but i use the variable outputs from the "Run" function (which are the names of the labels) into the "Voice" function, so it runs incredibly slow, it looks like fps equals 1
![image](https://user-images.githubusercontent.com/86045192/145395265-485d46af-b5fc-422a-bdf8-20c3c0c04f6a.png)
![image](https://user-images.githubusercontent.com/86045192/145395329-91e32fb7-f061-426f-8147-6c19dc884320.png)

So what i need to do to improve frame rate, Sir

thanks for your reply

**Comment by glenn-jocher** [comment]
@TuyenSD Ultralytics does not provide personal support for code modifications but perhaps a community member might have some feedback.
